### ISSUE TITLE
ETHTOOL_OPTIONS doesn't work

### ISSUE THREAD

**Issue by matwey** [openSUSE/wicked#685]
Hello,

I am using wicked-0.6.39 on openSUSE Leap 42.2.
I configured an interface as the following:

```# cat /etc/sysconfig/network/ifcfg-eth1 
BOOTPROTO='static'
BROADCAST=''
ETHTOOL_OPTIONS='-C eth1 rx-usecs 0'
IPADDR='10.1.0.1/24'
MTU=''
NAME='I210 Gigabit Network Connection'
NETWORK=''
REMOTE_IPADDR=''
STARTMODE='auto'
```

After system reboot, the interface started and running. However, ethtool options have not been applied on boot.

```
# ethtool -c eth1
Coalesce parameters for eth1:
Adaptive RX: off  TX: off
stats-block-usecs: 0
sample-interval: 0
pkt-rate-low: 0
pkt-rate-high: 0

rx-usecs: 3
rx-frames: 0
rx-usecs-irq: 0
rx-frames-irq: 0

tx-usecs: 0
tx-frames: 0
tx-usecs-irq: 0
tx-frames-irq: 0

rx-usecs-low: 0
rx-frame-low: 0
tx-usecs-low: 0
tx-frame-low: 0

rx-usecs-high: 0
rx-frame-high: 0
tx-usecs-high: 0
tx-frame-high: 0
```

At the same time, I believe it should say ```rx-usecs: 0```.
```
# ethtool -C eth1 rx-usecs 0
# ethtool -c eth1
Coalesce parameters for eth1:
Adaptive RX: off  TX: off
stats-block-usecs: 0
sample-interval: 0
pkt-rate-low: 0
pkt-rate-high: 0

rx-usecs: 0
rx-frames: 0
rx-usecs-irq: 0
rx-frames-irq: 0

tx-usecs: 0
tx-frames: 0
tx-usecs-irq: 0
tx-frames-irq: 0

rx-usecs-low: 0
rx-frame-low: 0
tx-usecs-low: 0
tx-frame-low: 0

rx-usecs-high: 0
rx-frame-high: 0
tx-usecs-high: 0
tx-frame-high: 0
```

How could I find what is going wrong here?

**Comment by mtomaschewski** [comment]
Yes, the code is a bit outdated and does not support ring or coalesce. We are working to add the unhandled options already (bsc#1007909,bsc1012366).

**Comment by matwey** [comment]
I don't have access both to 1007909 and 1012366. They seems to be for Enterprise users only.

**Comment by mtomaschewski** [comment]
fixed in 0.6.40 (referenced in changelog as [bsc#1007909](https://bugzilla.suse.com/show_bug.cgi?id=1007909))
